src/app/hooks/keycloak.ts

```diff
@@ -39,12 +44,25 @@ export const useAuth = (): Auth => {
     return instance;
   }, [config, insightsChromeAuth]);
 
+  useEffect(() => {
```

Comment:
To make it clear. We only need to call this API once as it is not changing and doesn't require refresh like other situations where we want the most recent version of the data.

Reply:
yes, it's calling only once as useEffect hook doesn't have dependency on any value. 
Another place to move it in  `AppEntry.tsx` file where we are passing auth object to context provider.

@riccardo-forina Any thought/suggestion on this ?

Reply:
I would prevent calling this endpoint many times due to useAuth being used in couple places (with different dependencies). My overall take is that this is too low level to be used across so many services. We need to have useSDK or something like this or even expose individual hooks (preffered). Scope for this PR is to ensure that we call this API only once when we move across different components.

Reply:
I've moved this logic into AppEbtry.tsx file.
As we are planning to decouple ui-shared auth object . I created local auth object in app-services-ui and passing values as pros.

I think we don't need to create separate auth object for each federated modules like kas, kafka, srs etc. We can have simple properties as we needed like getToke, getMASSOToken etc.

We will do rest of the decoupling changes in later PRs.

Reply:
> We can have simple properties as we needed like getToke, getMASSOToken etc

Sounds good. Eventually it will be just matter of using insights token. I think that auth should come from insights rather than us